README.md

```diff
@@ -16,6 +16,12 @@ easy.  Just do the following:
 7. Run `./manage.py runserver`
 8. Open a browser and point it at http://localhost:8000/
 
+## Docker Version
+
+1. docker build . -t ripe-atlas-halo
+1. docker run --rm -d -P ripe-atlas-halo
+1. Open a browser and point it at http://localhost:8000/
```

Comment:
This didn't work for me, since the `-P` above didn't allocate port 8000 but rather a random port.  I don't know how to get docker to assign a specific port, but if you do, then I'd recommend adding that above.  If not, then maybe this line should say something like *"Run `docker ps` to get the local port number and then open that in a browser"*?